storage/postgres/plugin.go

```diff
@@ -112,13 +137,21 @@ func (p *PostgresBackend) CreatePlugin(ctx context.Context, pluginDto types.Plug
 	var createdId string
 	err := p.pool.QueryRow(ctx, query, args).Scan(&createdId)
 	if err != nil {
-		return nil, err
+		return types.PluginDto{}, err
+	}
+
+	ratingQuery := `INSERT INTO plugin_rating (plugin_id, rating, count) 
```

Comment:
maybe better to have this into a separate repository for plugin_rating and have a service layer function calling 

PluginRepo.CreatePlugin(...)
PluginRatingRepo.CreatePluginRating(...)

let's have the reviews also in a separate repo as we threat them as an entity of their own